content/applications/hr/appraisals.rst

```diff
@@ -0,0 +1,479 @@
+==========
+Appraisals
+==========
+
+Odoo's *Appraisals* application is where employees are evaluated on a regular basis. Managers
+evaluate the performance of their employees, and employees can do a self-assessment. Appraisals are
+customizable and can be set for any kind of schedule.
+
+Appraisals give employees valuable feedback including goals to work towards and skills to improve
+upon. Appraisals may form the basis for raises, promotions, and other benefits. Regular appraisals
+are good for both the employees and the company, by letting employees know how they are performing,
+and how they can improve.
+
+Configuration
+=============
+
+The configuration menu is where the appraisal feedback template is edited, the frequency is set,
+the evaluation scale is managed, and where the data for the 360 feedback is stored.
+
+Settings
+--------
```

Comment:
The logic I used to format the doc, I have the main menu items as the main 'sections'. Configuration is the main title, since it is a top menu item. Each sub-head directly beneath the main menu items list out all the items you find within that menu. So if you click Configuration, you see Settings, Evaluation Scale, 360 Feedback, and Tags. Those are the 4 main sections in Configuration since that's how the menu is organized. The other main sections also match the top menu, so there is an Appraisals, Goals, and Reporting as well, and the sub-sections are what appears beneath those. This instance it is less obvious because Appraisals and Goals do not have anything beneath them, so those sections jus dive right into what those are/how they are used. Reports is the only other one, along with Configuration, that has a sub-menu.  I like to have the sections mirror the UI since that is intuitive to me. I hope that explains my logic as to why I organized the doc the way I did. If you have any questions, let me know!

Also, I put Configuration first since you need to make the configurations to understand how the settings affect everything before you go right in and do an appraisal. So that being first is the real wrench in the structure, visually.